src/test/java/ru/yandex/CourierAuthorizationWithoutAnyRequiredFieldTest.java

```diff
@@ -0,0 +1,77 @@
+package ru.yandex;
+
+import io.restassured.response.ValidatableResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class CourierAuthorizationWithoutAnyRequiredFieldTest {
+
+    String login;
+    String password;
+    int courierId;
+
+    @Before
+    public void setUp() {
+        CourierClient courierClient = new CourierClient();
+        RandomValuesGenerator randomValuesGenerator = new RandomValuesGenerator();
+        login = randomValuesGenerator.getRandomLogin();
+        password = randomValuesGenerator.getRandomPassword();
+        CourierCredentials credentials = new CourierCredentials(login, password);
+        courierClient.registerCourier(credentials);
+        ValidatableResponse responseLogin = courierClient.login(credentials);
+        courierId = responseLogin.extract().body().path("id");
+    }
+
+    @After
+    public void tearDown() {
+        CourierClient courierClient = new CourierClient();
+        courierClient.delete(courierId);
+    }
+
+    @Test
+    public void authorizationWithoutLoginTest() {
+        CourierClient courierClient = new CourierClient();
+        CourierCredentials credentials = new CourierCredentials();
+        credentials.setPassword(password);
+        ValidatableResponse loginResponse = courierClient.login(credentials);
+        int responseCode = loginResponse.extract().statusCode();
+        String responseString = loginResponse.extract().body().path("message");
+        String expectedString = "Недостаточно данных для входа";
+
+        assertThat("Должен вернуться код ответа 400", responseCode, equalTo(SC_BAD_REQUEST));
+        assertThat("В теле ответа должно быть сообщение (Недостаточно данных для входа)", responseString, equalTo(expectedString));
+
+    }
+
+    @Test
+    public void authorizationWithoutPasswordTest() {
+        CourierClient courierClient = new CourierClient();
+        CourierCredentials credentials = new CourierCredentials();
+        credentials.setLogin(login);
+        ValidatableResponse loginResponse = courierClient.login(credentials);
+        int responseCode = loginResponse.extract().statusCode();
+        String responseString = loginResponse.extract().body().path("message");
+        String expectedString = "Недостаточно данных для входа";
+
+        assertThat("Должен вернуться код ответа 400", responseCode, equalTo(SC_BAD_REQUEST));
+        assertThat("В теле ответа должно быть сообщение (Недостаточно данных для входа)", responseString, equalTo(expectedString));
+    }
+
+    @Test
+    public void authorizationWithoutLoginAndPasswordTest() {
```

Comment:
Твой тест падает с очень не хорошей ошибкой:
_Cannot invoke the path method because no content-type was present in the response and no default parser has been set.
You can specify a default parser using e.g.:
RestAssured.defaultParser = Parser.JSON;_
Со стороны это выглядит как косяк тестов, а не косяк сервиса. А это не так. Такая ситуация происходит из-за того, что наш сервер виснет не отдает тело ответа, но зато он отдает статус код. Если проверка на статус код делалась бы раньше проверки body, то тест бы падал с красивой ошибкой статус кода и вопросов бы не было. 
Поэтому я всегда рекомендую первым делом проверять статус код, а потом уже все остальное.